cli.js

```diff
@@ -12,40 +13,46 @@ const cli = meow(`
     --useColorArtboards             Use artboard formatting to export colors instead of using the document colors
     --useGradientArtboards          Use artboard formatting to export gradients instead of using the document gradients
     --useStyleDictionaryOutput, -s  Generate Style Dictionary compatible output instead of the generic design token format.
+    --token, -t                     Authorization token when accessing the Figma API
```

Comment:
Just a detail, but in these help examples please make sure the sentences end with a dot.